collections/ansible_collections/vmware/nsx_alb/docs/avi_actiongroupconfig.rst

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/python3
+#
+# @author: Gaurav Rastogi (grastogi@avinetworks.com)
+#          Eric Anderson (eanderson@avinetworks.com)
+# module_check: supported
+#
+# Copyright: (c) 2017 Gaurav Rastogi, <grastogi@avinetworks.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+#
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
```

Comment:
rst is not a yaml format Please check : https://raw.githubusercontent.com/ansible-collections/community.vmware/main/docs/community.vmware.vca_fw_module.rst